src/js/angular/security/controllers.js

```diff
@@ -931,6 +931,24 @@ securityCtrl.controller('ChangeUserPasswordSettingsCtrl', ['$scope', 'toastr', '
             ThemeService.applyTheme(theme.name);
         };
 
+        $licenseService.checkLicenseStatus().then(() => {
```

Comment:
Can you move this in the init or put it in a method and call it from the init

Reply:
Also catch is missing

Reply:
done